app/assets/stylesheets/ama_layout/layout_components/error-messages.scss

```diff
@@ -1,10 +0,0 @@
-.notice_notification{
```

Comment:
Heads up to @sineade and @kathleenAMA - this file gets changed in the `registries` branch https://github.com/amaabca/ama_layout/compare/registries#diff-cb69a10c988591d1e7df02fb1414c86bR11

so we'll have a merge conflict when we try to merge registries in